Some tests fail on Python 3.13: https://paste.fliegendewurst.eu/wybMOb.log
```
E       AssertionError: assert ['a/b', 'a/b/...'a/b/c/d.txt'] == ['a/b', 'a/b/c']
E         
E         Left contains one more item: 'a/b/c/d.txt'
E         Use -v to get more diff
tests/test_basic.py:87: AssertionError
=========================== short test summary info ============================
FAILED tests/test_basic.py::test_baseline_pathlib[a/b/c/d.txt;**] - AssertionError: assert ['a', 'a/b', ...'a/b/c/d.txt'] == ['a', 'a/b', 'a/b/c']
FAILED tests/test_basic.py::test_baseline_pathlib[a/b/c/d.txt;a/b/**] - AssertionError: assert ['a/b', 'a/b/...'a/b/c/d.txt'] == ['a/b', 'a/b/c']
=================== 2 failed, 143 passed, 2 skipped in 0.34s ===================
```
Could put these in disabledTests = lib.optionals pythonAtLeast ...

--- a/pkgs/development/python-modules/virtual-glob/default.nix
+++ b/pkgs/development/python-modules/virtual-glob/default.nix
@@ -1,0 +1,49 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  flit-core,
+  lib,
+  pytest,
+  pytestCheckHook,
+  pythonAtLeast,
+}:
+
+buildPythonPackage rec {
+  pname = "virtual-glob";
+  version = "0.2.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "chrisjsewell";
+    repo = "virtual-glob";
+    tag = "v${version}";
+    hash = "sha256-ocCa8m7mPPvzOZHPrraSEdSJZwRJoYO/Q7nyDbhIFu8=";
+  };
+
+  build-system = [
+    flit-core
+  ];
+
+  optional-dependencies = {
+    testing = [
+      pytest
+    ];
+  };
+
+  pythonImportsCheck = [
+    "virtual_glob"
+  ];
+
+  disabledTests = lib.optionals (pythonAtLeast "3.13") [
+    "test_baseline_pathlib"
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  meta = {
+    description = "Globbing of virtual file systems";
+    homepage = "https://pypi.org/project/virtual_glob/";
+    maintainers = with lib.maintainers; [ PopeRigby ];
+    license = lib.licenses.mit;
+  };
+}

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -17699,6 +17699,8 @@ self: super: with self; {
 
   virtkey = callPackage ../development/python-modules/virtkey { };
 
+  virtual-glob = callPackage ../development/python-modules/virtual-glob { };
+
   virtualenv = callPackage ../development/python-modules/virtualenv { };
 
   virtualenv-clone = callPackage ../development/python-modules/virtualenv-clone { };

